### PR TITLE
Feature/vehicle attachments

### LINKED ISSUE #558: Should announcement filters affect IRC?
Hey @OttoRocket-LVP -

I was very confused just now why my admin notice wasn't showing up on IRC, while it did show in-game. Tracing the code, I found that a condition in `announce_manager.js` blocked it.

I've re-enabled it for my own sake, as I often use this for testing: https://github.com/LVPlayground/playground/commit/e90a2bed601081b945233872c565d9f98a22b6e3

Let's use this to discuss however. Would we ever want to _not_ send messages to IRC?

Since we can't personalise that per administrator, my guess is that you were thinking about a global "turn announcement on or off" switch. I wonder if those should be the default values of the per-player settings instead? I think of IRC kind of like an info-dump where we share everything.

Thanks,
Russell

### PR DESCRIPTION
**Pull request by OttoRocket-LVP**
Thank you for making a contribution to [Las Venturas Playground](https://sa-mp.nl/)!

Please take a moment to fill in the questions in this template, to make it easier for our developers
to understand what you're doing.

## What is being changed?
_Allow the usage of vehicle attachments. This includes the DFT-40 functionality.
Also fix remove the PlayerSettings from Settings as per #558_

## Why is this being changed?
_To get rid of our FilterScript and have it in the gamemode. Also make it eas to add new vehicle customizations._

## How are you making the change?
  [X] I have a working check-out of Las Venturas Playground.
  [X] I have included tests for any changed JavaScript code.
  [X] I have tested this change myself on my local server.

### CODE REVIEW

File: javascript/features/settings/settings.js [hunk 1]
```diff
@@ -33,11 +33,6 @@ class Settings extends Feature {
         for (const setting of SettingList)
             this.settings_.set(setting.identifier, setting);
 
-        // Import player specific settings to also work server wide.
```
Comment (RussellLVP) [inline javascript/features/settings/settings.js:36]: Is this related to your change?

**Review by RussellLVP — APPROVED** [review]
Sure, thanks :)

File: javascript/features/settings/settings.js (hunk 1 above)
Comment (OttoRocket-LVP) [reply javascript/features/settings/settings.js:36]: Not really.

I updated the PR subscription. I'll make a seperate PR next time.
I came past this code thus took it with.
